const get = (req, res, next) => {
  try {
    throw new Error('sรณ foi');
  } catch (err) {
    next(err);
  }
};
export default get;
